plugin.yml

```diff
@@ -13,7 +13,8 @@ awareness:
 - !@UTF8
 
 depend: []
-softdepend: []
+softdepend:
+  - Factions
```

Comment:
Not having a softdepend on factions was intentional, it caused some kind of dependency loop. See https://github.com/MinecraftWars/Gringotts/commit/7bc31b6bd2894587ba95a407b1f75021d5525ce0#diff-5df3d3f42757c737a95531b868d1d2b9